### PR TITLE
Improve help text

### PR DESCRIPTION
**Pull request by wallyworld**
Fixes: https://bugs.launchpad.net/juju/+bug/1630863
Fixes: https://bugs.launchpad.net/juju/+bug/1631199

Improve help text for list-users command.
Fix help text in upgrade-juju command.

### CODE REVIEW

File: cmd/juju/user/list.go [hunk 1]
```diff
@@ -22,13 +22,22 @@ import (
 )
 
 var usageListUsersSummary = `
-Lists Juju users allowed to connect to a controller.`[1:]
+Lists Juju users allowed to connect to a controller or model.
+When used without a model name argument, users relevant to a controller are printed.
```
Comment (mjs) [inline cmd/juju/user/list.go]: Aren't these supposed to be a single line?

Comment (wallyworld) [inline cmd/juju/user/list.go]: fixed

File: cmd/juju/user/list.go [hunk 2]
```diff
@@ -22,13 +22,21 @@ import (
 )
 
 var usageListUsersSummary = `
-Lists Juju users allowed to connect to a controller.`[1:]
+Lists Juju users allowed to connect to a controller or model.`[1:]
 
 var usageListUsersDetails = `
-By default, the tabular format is used.
+When used without a model name argument, users relevant to a controller are printed.
+When used with a model name, users relevant to the specified model are printed.By default, the tabular format is used.
```
Comment (mjs) [inline cmd/juju/user/list.go]: Line wrapping?

**Comment by wallyworld** [comment]
$$merge$$

**Pull request merged**